src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -50,6 +50,24 @@ ucs_config_field_t uct_ib_mlx5_iface_config_table[] = {
      ucs_offsetof(uct_ib_mlx5_iface_config_t, mmio_mode),
      UCS_CONFIG_TYPE_ENUM(uct_ib_mlx5_mmio_modes)},
 
+    {"AR_ENABLE", "try",
+     "Enable Adaptive Routing (out of order) feature on SL that supports it.\n"
+     "SLs are selected as follows:\n"
+     "+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
```

Comment:
is it printed as one line?

Reply:
> is it printed as one line?

yes, do you need a human-readable format? :)

Reply:
```
#
# Enable Adaptive Routing (out of order) feature on SL that supports it.
# SLs are selected as follows:
# +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
# +                                         + UCX_IB_AR=yes         + UCX_IB_AR=no          + UCX_IB_AR=try         +
# +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
# +                + AR enabled on some SLs + Use 1st SL with AR    + Use 1st SL without AR + Use 1st SL with AR    +
# +                ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
# + UCX_IB_SL=auto + AR enabled on all SLs  + Use SL=0              + Failure               + Use SL=0              +
# +                ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
# +                + AR disabled on all SLs + Failure               + Use SL=0              + Use SL=0              +
# +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
# +                + AR enabled on <sl>     + Use SL=<sl>           + Failure               + Use SL=<sl>           +
# + UCX_IB_SL=<sl> ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
# +                + AR disabled on <sl>    + Failure               + Use SL=<sl>           + Use SL=<sl>           +
# +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
#
#
# syntax:    <yes|no|try>
# inherits:  UCX_RC_AR_ENABLE, UCX_IB_AR_ENABLE
#
UCX_RC_MLX5_AR_ENABLE=try
```